docs/data-sources/dbsnp-json.md

```diff
@@ -0,0 +1,8 @@
+```json
+"dbsnp":[
+   "rs1042821"
+]
+```
+| Field            | Type         | Required | Notes                     |
+|:-----------------|:------------:|:--------:|:--------------------------|
```

Comment:
Even though we had `Required` in our previous documentation and in ClinVar. We might want to consider removing it since there's often confusion about what it means.

Reply:
done